cmd/web/main.go

```diff
@@ -499,10 +499,7 @@ WHERE p.id = ?1`
 		return
 	}
 
-	lang, safe := enry.GetLanguageByFilename(fname)
-	if lang == "" || !safe {
-		lang, _ = enry.GetLanguageByContent(fname, []byte(data))
-	}
+	lang := enry.GetLanguage(fname, []byte(data))
```

Comment:
Does this also resolve the hack for markdown below? Assuming one of the strategies used by `GetLanguage` is by extension it should.